src/main/java/org/threadly/concurrent/CentralThreadlyPool.java

```diff
@@ -0,0 +1,215 @@
+package org.threadly.concurrent;
+
+import java.util.concurrent.atomic.LongAdder;
+
+import org.threadly.concurrent.wrapper.PrioritySchedulerDefaultPriorityWrapper;
+import org.threadly.concurrent.wrapper.limiter.SchedulerServiceLimiter;
+import org.threadly.util.ExceptionUtils;
+
+/**
+ * Threadly's centrally provided pool manager.  This class is designed to avoid needing to 
+ * manage the thread pool lifecycle throughout your application.  Instead of needing to think about 
+ * how to manage a pools lifecycle you can instead just describe the needs of the tasks to be 
+ * submitted.
+ * <p>
+ * Internally this will delegate to a central or otherwise specific pool for your needs, while 
+ * minimizing thread creation / churn for you.  In addition the returned pools do not need to be 
+ * shutdown, but instead you can allow to be garbage collected as you are done with them. 
+ * <p>
+ * Unlike other pools, you can schedule a task out on pools returned from this one and then allow 
+ * it to be garbage collected.  The scheduled task will run even if you no longer have a reference 
+ * to the original pool.
+ * 
+ * @since 5.4
+ */
+public class CentralThreadlyPool {
+  protected static final int LOW_PRIORITY_MAX_WAIT_IN_MS = 1000;  // TODO - configurable?  Diff from default?
+  protected static final PriorityScheduler MASTER_SCHEDULER;
+  protected static final PrioritySchedulerService LOW_PRIORITY_MASTER_SCHEDULER;
+  protected static final SchedulerService COMPUTATION_POOL;
+  protected static final SchedulerService LOW_PRIORITY_POOL;
+  protected static final PrioritySchedulerService SINGLE_THREADED_LOW_PRIORITY_POOL;
```

Comment:
I personally would also like to see a centeral Keyed scheduler/executor.  I would use that alot honestly.

Reply:
Agree. That is one important feature of CTP I expected.

Reply:
I considered having one, but the single thread scheduler design kind of overtook it.  Let me know your thoughts on if it would be a better option?  What would the advantage be of a global keyed executor?

Reply:
I think the benefits of keyed are:

- Config or Show infomation of Executors managed by CTP. Manageable is very important to online system. I'd really like to change some config without restart server. Of course, check CTP information is important as well.
- I might wanted to temproarily increase priority of Executor by its name

Reply:
@jiming I talked more about this with @lwahlmeier directly.  I think part of the problem is we are trying to solve different things (and that's in part what makes this so hard).

@lwahlmeier was thinking that this could become a replacement for all pools within a service.  And while I guess that's technically possible, that's not what I was thinking.

I was thinking this really would be useful for replacing small one off pools.  Things like "I need a cleanup task for this class", or "I want to report metrics / something".  I would think things like request pools, or pools which are heavily active / important to the service, would not be replaced by this.  I personally would not want to give up the control, or the performance sacrifices in replacing such important pools with something like this.

What were you thinking this role would be?  Did you want it to replace all pooling, or just common / small, utility pools?

Reply:
@jentfoo 

Now I can understand your means. 

Maybe next version, you could add some more features. My options is that you already go this far. Why not go a little further? Make threadly a de facto for ThreadPool Manager in JDK.

> What were you thinking this role would be? Did you want it to replace all pooling, or just common / small, utility pools?

I like a ONE who rule them all Thread Pool manager?

Thanks!